ksql-serde/src/test/java/io/confluent/ksql/serde/avro/KsqlGenericRowAvroSerializerTest.java

```diff
@@ -41,35 +43,35 @@
 
 public class KsqlGenericRowAvroSerializerTest {
 
-
-  private org.apache.kafka.connect.data.Schema schema;
-
-  @Before
-  public void before() {
-
-    schema = SchemaBuilder.struct()
-        .field("ordertime".toUpperCase(), org.apache.kafka.connect.data.Schema.INT64_SCHEMA)
-        .field("orderid".toUpperCase(), org.apache.kafka.connect.data.Schema.INT64_SCHEMA)
-        .field("itemid".toUpperCase(), org.apache.kafka.connect.data.Schema.STRING_SCHEMA)
-        .field("orderunits".toUpperCase(), org.apache.kafka.connect.data.Schema.FLOAT64_SCHEMA)
-        .field("arraycol".toUpperCase(), SchemaBuilder.array(org.apache.kafka.connect.data.Schema.FLOAT64_SCHEMA))
-        .field("mapcol".toUpperCase(), SchemaBuilder.map(org.apache.kafka.connect.data.Schema.STRING_SCHEMA, org.apache.kafka.connect.data.Schema.FLOAT64_SCHEMA))
+  final Schema schema = SchemaBuilder.struct()
+        .field("ordertime".toUpperCase(), Schema.INT64_SCHEMA)
+        .field("orderid".toUpperCase(), Schema.INT64_SCHEMA)
+        .field("itemid".toUpperCase(), Schema.STRING_SCHEMA)
+        .field("orderunits".toUpperCase(), Schema.FLOAT64_SCHEMA)
+        .field("arraycol".toUpperCase(), SchemaBuilder.array(Schema.FLOAT64_SCHEMA).build())
+        .field(
+            "mapcol".toUpperCase(),
+            SchemaBuilder.map(Schema.STRING_SCHEMA, Schema.FLOAT64_SCHEMA).build())
```

Comment:
Optional?

Reply:
The engine layer doesn't set these optional (as of this PR). Leaving this not-optional ensures that the serde handles the schema provided by the engine correctly. Again, can change once your change to make everything optional goes in.